### DISCUSSION TITLE
Exclude a column from being fetched at all?

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#9703] [Help]
Currently Filament sends the entire rows JSON to the /edit page. I have one column with binary data in this table and it's making Filament crap itself. Page doesn't load with a ton of JS errors in console.

Now this could be a bug and might be fixed, but until then how do I make it so it doesn't fetch this data?

**Comment by sinnbeck** [comment]
You could add this to the resource
```php
 public static function getEloquentQuery(): Builder
    {
        return parent::getEloquentQuery()
            ->select('id', 'name', 'foo');
    }
